internal/server/disk/internal/types.go

```diff
@@ -35,22 +35,54 @@ type GetDiskNumberByNameRequest struct {
 }
```

Comment:
put more details comments on DiskName, it is kind of confusing with all these number, id , name etc ..., unfortunately windows has so many different ways to identify a disk..

Reply:
Update to:

```
type GetDiskNumberByNameRequest struct {
	// Disk name is the page83 ID of the disk
	DiskName string
}
```